kubernetes/config-maps/schedule-configmap.yaml

```diff
@@ -16,4 +16,5 @@ data:
   publisher.ts.private.2018: 0 30 15 ? * SUN *
   publisher.ts.private.2019: 0 10 6 * * ?
   publisher.ts.public.2018: 0 30 10 ? * SUN *
+  publisher.ts.public.2018: 0 30 12 ? * SUN *
```

Comment:
This line seems to be a duplicate. Lets keep either `0 30 10 ? * SUN *` or `0 30 12 ? * SUN *`